oasis-webapp/src/main/resources/templates/ChangePassword.soy

```diff
@@ -35,19 +35,22 @@ h1 { font-weight: lighter; font-size: 1.8em; }
 button, a.portal {
   margin: 5px 1em;
   padding: .5em 2em;
-  border: 1px solid #6e418e;
-  border-radius: 10px;
   cursor: pointer;
   text-decoration: none;
   display: inline-block;
+  border: 1px solid #4c2d62;
+  color: #fff;
```

Comment:
`background` is not defined here, and `color` is overridden in both cases below; so `color` is unnecessary here.